### PR TITLE
Use setup.cfg for package metadata (ref #1921)

### PR DESCRIPTION
**Pull request by leplatrem**
Ref #1921

IMO it makes more sense to have package metadata in a configuration file rather than an executed python file.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,81 @@
+[metadata]
+name = kinto
+version = 13.1.0.dev0
```
Comment (Natim) [inline setup.cfg]: Do you know if zest.releaser handle this?

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+description = Kinto Web Service - Store, Sync, Share, and Self-Host.
+long_description = file: README.rst, CHANGELOG.rst, CONTRIBUTORS.rst
```
Comment (Natim) [inline setup.cfg]: :1st_place_medal:

File: setup.cfg (hunk 1 above)
Comment (Natim) [reply setup.cfg]: :heart_decoration:

File: setup.cfg [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+license = Apache License (2.0)
+author = Mozilla Services
+author_email = storage-team@mozilla.com
+url = https://github.com/Kinto/kinto
+download_url = https://github.com/dj-stripe/dj-stripe/tarball/master
```
[the remark this answers is not in the document]
Comment (leplatrem) [reply setup.cfg]: Thanks, fixed in 80c4a7d18

**Pull request merged**